tests/qit/phpstan.sh

```diff
@@ -6,9 +6,33 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Source common.sh using the relative path
 source "$DIR/common.sh"
 
-echo "Running PHPStan tests..."
-$QIT_BINARY run:phpstan woocommerce-payments --wait
-if [ $? -ne 0 ]; then
-	echo "Failed to run PHPStan command. Exiting with status 1."
-	exit 1
+# Check if the --local flag is provided which means the tests should run against the development build
+LOCAL_FLAG=false
+ZIP_FILE=""
+
+for arg in "$@"; do
+  if [[ "$arg" == "--local" ]]; then
+    LOCAL_FLAG=true
+  fi
+done
+
+if [[ "$LOCAL_FLAG" == true ]]; then
```

Comment:
```suggestion
ZIP_FILE=""

if echo "$@" | grep -q -- "--local"; then
```

(⛏️ ) What do you think about this approach for checking for the `--local` flag?

Reply:
Sounds good, simplified it in 1197c5dd38a1a1e32fc8091dfafedf7ce4394137, thanks!